src/transcriptProcessing/transcriptToCuts.ts

```diff
@@ -1,16 +1,27 @@
 import { updateOutputTimes } from './updateOutputTimes';
-import { Transcription, Cut } from '../sharedTypes';
+import { Transcription, Cut, IndexRange } from '../sharedTypes';
 import {
   bufferedWordDuration,
   roundToMs,
   isInInactiveTake,
 } from '../sharedUtils';
 
-const convertTranscriptToCuts = (transcript: Transcription): Array<Cut> => {
-  // filter out deleted words as well as words in inactive takes
-  const wordsNotYetUpdated = transcript.words.filter(
-    (word) => !word.deleted && !isInInactiveTake(word, transcript.takeGroups)
-  );
+const convertTranscriptToCuts = (
+  transcript: Transcription,
+  rangeOverride: IndexRange | null = null
+): Array<Cut> => {
+  const wordsNotYetUpdated =
+    // filter out deleted words as well as words in inactive takes
+    rangeOverride === null
+      ? transcript.words.filter(
+          (word) =>
+            !word.deleted && !isInInactiveTake(word, transcript.takeGroups)
+        )
+      : // keep deleted words when rangeOverride is specified
+        transcript.words.slice(
```

Comment:
ah cool so we can use rangeOverride for take detection too if needed. Nice!

Reply:
wdym by this? currently only used for previewing deleted sections but yeah it was architected so that it's flexible-ish

Reply:
the plan later down the track for take detection is to have preview of the take when selecting takes. This could be used for that

Reply:
Yep so that would be able to reuse like half this code - basically video preview now supports either previewing the entire video or a fixed range of words

Reply:
Yep so that would be able to reuse like half this code - basically video preview now supports either previewing the entire video or a fixed range of words